### PR TITLE
71 siteadmin touch ui debug

### PR DESCRIPTION
**Pull request by kkarolk**

### CODE REVIEW

File: bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/SiteadminPage.java [hunk 1]
```diff
@@ -102,11 +107,30 @@ public SiteadminActions copyPage(String title, String destination) {
...
     toolbar.copyPage();
     open(destination);
+    int pageCount = getChildPageWindow().getPageCount();
     toolbar.pastePage();
+    waitForPageCount(pageCount + 1);
     waitForPageToAppearOnTheList(title);
     return this;
   }
 
+  public SiteadminPage waitForPageCount(int pageCount) {
```
Comment (kkarolk) [inline bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/SiteadminPage.java]: this may be added to the interface?

Comment (youngeagle87) [inline bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/SiteadminPage.java]: I agree

File: bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/list/ChildPageWindow.java [hunk 2]
```diff
@@ -56,11 +66,23 @@ public boolean isLoaded() {
   }
 
   public boolean containsPage(String title) {
-    return childPages.stream().filter(t -> t.getTitle().equals(title)).findFirst().isPresent();
+    return webElementUtils.isConditionMet(new ExpectedCondition<Object>() {
+      @Nullable @Override public Object apply(@Nullable WebDriver input) {
```
Comment (youngeagle87) [inline bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/list/ChildPageWindow.java]: webdriver name

**Comment by youngeagle87** [comment]
Added improvements. Approve

**Pull request merged**